lib/config/default.js

```diff
@@ -162,5 +162,6 @@ module.exports = {
   allowEmailRegister: true,
   allowGravatar: true,
   allowPDFExport: true,
-  openID: false
+  openID: false,
+  linkifyHeaderStyle: 'keep-case'
```

Comment:
Hi @hoijui 
Could you add a comment in here to describe the possible options?
We will also add it into the documentation.

Reply:
yes!

Reply:
done here:
https://github.com/hackmdio/codimd/pull/1310/commits/3d5d8e20f73bf8d363975273f8ade70d729aa126